services/node/node-lts.tester.js

```diff
@@ -0,0 +1,166 @@
+'use strict'
+
+const { expect } = require('chai')
+const { Range } = require('semver')
+const t = (module.exports = require('../tester').createServiceTester())
+const {
+  mockPackageData,
+  mockNonExistingPackageData,
+  mockReleaseSchedule,
+  mockVersionsSha,
+} = require('./testUtils/test-utils')
+
+function expectSemverRange(message) {
+  expect(() => new Range(message)).not.to.throw()
+}
+
+t.create('engines satisfies all lts node versions')
+  .get('/passport.json')
+  .intercept(mockReleaseSchedule())
+  .intercept(mockPackageData(`passport`, `10 - 12`))
+  .intercept(mockVersionsSha())
+  .expectBadge({ label: 'node lts', message: `10 - 12`, color: `brightgreen` })
+  .afterJSON(json => {
+    expectSemverRange(json.message)
+  })
```

Comment:
Do we need this for these mocked tests given that `expectBadge` is already doing an exact match on the expected message?

Reply:
Nope, I will remove them in mocked tests